src/main/groovy/org/pih/warehouse/core/HistoryItem.groovy

```diff
@@ -10,4 +10,8 @@ class HistoryItem<T> {
     ReferenceDocument referenceDocument
 
     EventCode eventCode
+
+    Comment comment
+
+    User createdBy
```

Comment:
(my bad) I can see redundant `identifier` in this domain (it was moved to the `ReferenceDocument`).
Could you please remove the `identifier` from the `HistoryItem` and the assignments of it (Shipment domain, Receipt domain, Event domain).